projects/storefrontlib/src/cms-components/product/product-images/product-images.component.ts

```diff
@@ -22,19 +22,8 @@ export class ProductImagesComponent {
     )
   );
 
-  /**
-   * @deprecated since version 1.5
-   * This variable will no longer be in use. Use thumbs$ observable instead.
-   * TODO(issue:#6166).
-   */
-  isThumbsEmpty: boolean;
```

Comment:
This should be handled in update-schematics in two ways:
1. if customers are using the selector for this component, we should put a comment in their html template saying that this property has been removed. To do this, please follow the README instructions defined here https://github.com/SAP/cloud-commerce-spartacus-storefront/pull/6769 (not yet merged)
2. if customers extended this component and use `isThumbsEmpty`, they'll get an error. In this case we should also put a comment. To do this, you can use `projects/schematics/src/shared/utils/file-utils.ts#insertCommentAboveIdentifier`, but please wait for this PR https://github.com/SAP/cloud-commerce-spartacus-storefront/pull/6790, as it will bring some required change to the method.